webservices/resources/legal.py

```diff
@@ -227,7 +227,7 @@ def apply_ao_specific_query_params(query, **kwargs):
     citation_queries = []
     if kwargs.get('ao_regulatory_citation'):
         for citation in kwargs.get('ao_regulatory_citation'):
-            exact_match = re.match(r"(?P<title>\d+)\s+CFR\s+§*(?P<part>\d+)\.(?P<section>\d+)", citation)
+            exact_match = re.match(r"(?P<title>\d+)\s+C.*F.*R.*\s+§*(?P<part>\d+)\.(?P<section>\d+)", citation)
```

Comment:
Again, wouldn't this need to escape the `.` character so it matches it exactly?  `.` on its own means "match any one single character, regardless of what it is."

Reply:
Yes, thanks! I forgot to update this section.